.github/dependabot.yml

```diff
@@ -1,10 +1,15 @@
 # File generated by "make gendependabot"; DO NOT EDIT.
+
 version: 2
 updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "docker"
```

Comment:
Does this support looking for Dockerfiles under the specified directory only or its subdirectories as well?

Reply:
subdirectories as well.